HeartbeatLogging/Sources/StorageFactory.swift

```diff
@@ -16,37 +16,45 @@ import Foundation
 
 /// A factory type for `Storage`.
 protocol StorageFactory {
-  static func makeStorage(id: String) -> Self
+  static func makeStorage(id: String) -> Storage
 }
 
 // MARK: - FileStorage + StorageFactory
 
+/// The name of the file system directory where heartbeat data is stored.
+let kHeartbeatFileStorageDirectoryPath = "google-heartbeat-storage"
```

Comment:
From the Google Swift style guide https://google.github.io/swift/#global-constants we shouldn't be using a `k` prefix. I'm a fan of having a private `Constants` struct that has constants for this file so it's obvious at the call site where the variable is coming from. WDYT?

Reply:
Reworked in 29cf736